main.tf

```diff
@@ -259,6 +259,7 @@ resource "aws_ecs_service" "ignore_changes_task_definition" {
   platform_version                   = var.launch_type == "FARGATE" ? var.platform_version : null
   scheduling_strategy                = var.launch_type == "FARGATE" ? "REPLICA" : var.scheduling_strategy
   enable_ecs_managed_tags            = var.enable_ecs_managed_tags
+  iam_role                           = aws_iam_role.ecs_service.arn
```

Comment:
```suggestion
  iam_role                           = join("", aws_iam_role.ecs_service.*.arn)
```

because of 
```
TestExamplesComplete 2020-05-29T20:41:57Z command.go:121: Error: Missing resource instance key                                                                                           
TestExamplesComplete 2020-05-29T20:41:57Z command.go:121:                                                                                                                                
TestExamplesComplete 2020-05-29T20:41:57Z command.go:121:   on ../../main.tf line 262, in resource "aws_ecs_service" "ignore_changes_task_definition":                                   
TestExamplesComplete 2020-05-29T20:41:57Z command.go:121:  262:   iam_role                           = aws_iam_role.ecs_service.arn                                                      
TestExamplesComplete 2020-05-29T20:41:57Z command.go:121:                                                                                                                                
TestExamplesComplete 2020-05-29T20:41:57Z command.go:121: Because aws_iam_role.ecs_service has "count" set, its attributes must be                                                       
TestExamplesComplete 2020-05-29T20:41:57Z command.go:121: accessed on specific instances.                                                                                                
Te
```